rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -79,6 +79,41 @@ void fmt_class_string<sys_net_error>::format(std::string& out, u64 arg)
 	});
 }
 
+#ifdef _WIN32
+// Workaround function for WSAPoll not reporting failed connections
+void windows_poll(pollfd* fds, unsigned long nfds, int timeout, bool *connecting)
+{
+	const auto finished_connecting = [](SOCKET fd)
```

Comment:
Okay, now need to fix this because it's not possible to use idm with SOCKET. I suggest to write false back to connecting[i], and then read it on call site.